packages/loaders/soap/src/SOAPLoader.ts

```diff
@@ -556,7 +556,6 @@ export class SOAPLoader {
   getAliasMapFromAttributes(attributes: XSSchema['attributes'] | WSDLDefinition['attributes']) {
     const aliasMap = new Map<string, string>();
     for (const attributeName in attributes) {
-      // @ts-expect-error - Weird TS error
       const attributeValue = attributes[attributeName];
```

Comment:
Do we leave it as is ? Or should it be recorded in an issue to solve it later ?

Reply:
There is no error anymore. What should we record?

Reply:
That's weird 😅